### PR TITLE
ca65: added .P4510 and .IFP4510 pseudo commands

### PR DESCRIPTION
**Pull request by SvOlli**
ca65: added .P4510 and .IFP4510 analog to .PC02 and .IFPC02 commands
- added testcase to "test/assembler"
- updated documentation

getcpu.s: added detection for 4510 CPU

### CODE REVIEW

File: doc/ca65.sgml [hunk 1]
```diff
@@ -3805,6 +3827,7 @@ Here's a list of all control commands and a description, what they do:
        <tt><ref id=".IFPSC02" name=".IFPSC02"></tt>,
        <tt><ref id=".P02" name=".P02"></tt>,
        <tt><ref id=".P816" name=".P816"></tt>,
+       <tt><ref id=".P4510" name=".P4510"></tt>
```
Comment (greg-king5) [inline doc/ca65.sgml]: Typo: a comma is missing from the end of the line.

**Comment by SvOlli** [comment]
added the missing ",", since no other topic has arisen over the weekend

**Comment by greg-king5** [comment]
What are the problems that you had with `WORKDIR` in "test/assembler/Makefile" (what are the symptoms)?

**Comment by SvOlli** [comment]
Not everything could be put in workdir, *.o was still left in the same directory as *.s, so building stuff out of tree seemed rather useless, since cleaning up the test/assembler directory still needs to be done. The Makefile is also a bit more readable that way.

**Comment by oliverschmidt** [comment]
> *.o was still left in the same directory as *.s

See https://github.com/cc65/cc65/issues/168.

**Comment by SvOlli** [comment]
I am now watching #168 so I can re-add WORKDIR, once the issue is fixed.

**Comment by SvOlli** [comment]
I found a nice workaround: after building the assembler test target, the corresponding *.o file gets removed afterwards, so nothing has been left over. Also noted in a comment why this happens, so it can be removed once #168 is fixed. (Fixing #168 will not break this build, as the "-f" option to rm always returns 0, event if the file does not exist.)

**Comment by SvOlli** [comment]
Has this one "aged enough" without objections to be acceptable? ;-)

**Comment by oliverschmidt** [comment]
Sorry, I missed that you had answered Gregs question.

**Pull request merged**